pkg/trace/api/api.go

```diff
@@ -210,9 +218,20 @@ func (r *HTTPReceiver) replyTraces(v Version, w http.ResponseWriter) {
 
 // handleTraces knows how to handle a bunch of traces
 func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.Request) {
+	if atomic.LoadInt64(&r.refuse) == 1 {
+		// using too much memory
```

Comment:
Nit: that comment makes more sense at the definition of `refuse` or when you set it to 1. In the future, we might turn `refuse` on for different reasons while `handleTraces` will be unaffected.